src/Avalonia.Controls/Converters/PlatformKeyGestureConverter.cs

```diff
@@ -41,156 +41,7 @@ public class PlatformKeyGestureConverter : IValueConverter
-                s.Append("Alt");
-            }
-
-            if (gesture.KeyModifiers.HasAllFlags(KeyModifiers.Meta))
-            {
-                Plus(s);
-                s.Append(meta);
-            }
-
-            Plus(s);
-            s.Append(ToString(gesture.Key));
-
-            return StringBuilderCache.GetStringAndRelease(s);
-        }
-
-        private static string ToOSXString(KeyGesture gesture)
-        {
-            var s = StringBuilderCache.Acquire();
-
-            if (gesture.KeyModifiers.HasAllFlags(KeyModifiers.Control))
-            {
-                s.Append('⌃');
-            }
-
-            if (gesture.KeyModifiers.HasAllFlags(KeyModifiers.Alt))
-            {
-                s.Append('⌥');
-            }
-
-            if (gesture.KeyModifiers.HasAllFlags(KeyModifiers.Shift))
-            {
-                s.Append('⇧');
-            }
-
-            if (gesture.KeyModifiers.HasAllFlags(KeyModifiers.Meta))
-            {
-                s.Append('⌘');
-            }
-
-            s.Append(ToOSXString(gesture.Key));
-
-            return StringBuilderCache.GetStringAndRelease(s);
-        }
-
-        private static string ToString(Key key)
-        {
-            return key switch
-            {
-                Key.Add => "+",
-                Key.Back => "Backspace",
-                Key.D0 => "0",
-                Key.D1 => "1",
-                Key.D2 => "2",
-                Key.D3 => "3",
-                Key.D4 => "4",
-                Key.D5 => "5",
-                Key.D6 => "6",
-                Key.D7 => "7",
-                Key.D8 => "8",
-                Key.D9 => "9",
-                Key.Decimal => ".",
-                Key.Divide => "/",
-                Key.Down => "Down Arrow",
-                Key.Left => "Left Arrow",
-                Key.Multiply => "*",
-                Key.OemBackslash => "\\",
-                Key.OemCloseBrackets => "]",
-                Key.OemComma => ",",
-                Key.OemMinus => "-",
-                Key.OemOpenBrackets => "[",
-                Key.OemPeriod=> ".",
-                Key.OemPipe => "|",
-                Key.OemPlus => "+",
-                Key.OemQuestion => "/",
-                Key.OemQuotes => "\"",
-                Key.OemSemicolon => ";",
-                Key.OemTilde => "`",
-                Key.Right => "Right Arrow",
-                Key.Separator => "/",
-                Key.Subtract => "-",
-                Key.Up => "Up Arrow",
-                _ => key.ToString(),
-            };
-        }
-
-        private static string ToOSXString(Key key)
-        {
-            return key switch
-            {
-                Key.Back => "⌫",
-                Key.Down => "↓",
-                Key.End => "↘",
-                Key.Escape => "⎋",
-                Key.Home => "↖",
-                Key.Left => "←",
-                Key.Return => "↩",
-                Key.PageDown => "⇞",
-                Key.PageUp => "⇟",
```

Comment:
I don't use macOS/iOS, but I think `PageDown`/`PageUp` were swapped here (I found [this image](https://stackoverflow.com/q/61939452) on SO), so it should be fixed on the new locations:

- https://github.com/AvaloniaUI/Avalonia/pull/15828/files#diff-4c0ae86a03106d9faf050d2b7bb00c5b3a9700408e49eb2a400dc25459722ca8R130-R131
- https://github.com/AvaloniaUI/Avalonia/pull/15828/files#diff-22e337dc37b9ba4b40871c09bacc7275e05206e9fadd2d01e13cd9caf0c8a2a6R89-R90

Reply:
I did think that looked weird, and doing a bit of searching have found other references to the same.  I'll fix those up.